src/content/index.md

```diff
@@ -13,7 +13,7 @@ people we serve.</span>
 <column lg="16">
 
 <video>
-  <iframe title="IBM Design sizzle video" src="https://player.vimeo.com/video/304672438?muted=1&autoplay=1&loop=1?title=0&byline=0&portrait=0?color=ff0000" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen />
+  <iframe id="homepage-video" title="IBM Design sizzle video" src="https://player.vimeo.com/video/329866992?&muted=1&loop=0&autoplay=1" allow="autoplay" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen />
```

Comment:
@jnm2377 Wondering if putting this `<iframe>` into `<Video>` itself is an option - In this way we can easily grab the `<iframe>` without needing to set `id` here. Any thoughts?

Reply:
@asudoh like just removing `<iframe>` from the markdown and placing it where we currently render `{children}` within the component, and then select it like `document.querySelector('.ibm--video-wrapper iframe')`?

Reply:
@jnm2377 Yeah close - If `<Video>` renders the `<iframe>` then we can `<iframe ref={this.iframe}>`. `this.iframe` should be set at around L16 as `this.iframe = React.createRef();`. In this way, we can do `this.player = new Player(this.iframe.current)` at L23.

Reply:
Ah I see what you mean. I tried it out and it worked correctly, but the component itself was no longer rendering in a `gatsby-resp-iframe-wrapper` anymore, which altered the video size/responsive styles it was getting from that and making the `iframe` small.

Reply:
Got it - Seems that https://www.gatsbyjs.org/packages/gatsby-remark-responsive-iframe/ doesn't work if the `<iframe>` is in a component. In such case, we can grab the DOM ref of the container and search for the `<iframe>` in it, like:

```diff
--- a/src/components/Video/Video.js
+++ b/src/components/Video/Video.js
@@ -14,12 +14,14 @@ export default class Video extends React.Component {
     this.handlePlaySecondVideo = this.handlePlaySecondVideo.bind(this);
   }
 
+  _container = React.createRef();
+
   state = {
     isVideoWrapperActive: false,
   };
 
   componentDidMount() {
-    const iframe = document.querySelector('#homepage-video');
+    const iframe = this._container.current.querySelector('iframe');
     this.player = new Player(iframe);
     this.player.on('ended', this.onEnd);
     this.player.setLoop(false);
@@ -158,6 +160,7 @@ export default class Video extends React.Component {
 
     return (
       <div
+        ref={this._container}
         className={videoWrapperClassName}
         onClick={this.handlePlaySecondVideo}
         onKeyDown={this.handlePlaySecondVideo}
--- a/src/content/index.md
+++ b/src/content/index.md
@@ -13,7 +13,7 @@ people we serve.</span>
 <column lg="16">
 
 <video>
-  <iframe id="homepage-video" title="IBM Design sizzle video" src="https://player.vimeo.com/video/329866992?&muted=1&loop=0&autoplay=1" allow="autoplay" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen />
+  <iframe title="IBM Design sizzle video" src="https://player.vimeo.com/video/329866992?&muted=1&loop=0&autoplay=1" allow="autoplay" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen />
 </video>
 
 </column>
```